### ISSUE TITLE
[IMPROVEMENT] Combine App Icon/Color Scheme

### ISSUE THREAD

**Issue by soundstorm** [RocketChat/Rocket.Chat.iOS#1827]
We currently have two menu items for theming, one is the general color appearance and the other is the app icon. We may combine them into one.
Btw: do we really need an explanation for choosing an app icon?

**Comment by filipealva** [comment]
Hi @soundstorm, thanks for your feedback! It's this way because theming and changing the app icon are two different features. To address your suggestion we would have to think of cutting out the "change app icon feature" and adding up to the theming feature the ability to make the app icon follow the theme color scheme. We will surely consider it when discussing about the future of the theming feature.

About the explanation for choosing an app icon: we've decided to add it since we have a lot of non-tech-savvy users that not even know that it is possible to change an app's icon on iOS.

**Comment by filipealva** [comment]
@soundstorm yes it surely seems not wrong, but today the user can’t customize a single attribute of a theme, and we have not decided yet if it will be possible. If we combine theming with app icon changing we would change the app icon to dark but the user would be able to change it back to light even with the dark theme enabled, i.e. he would be able to change a single attribute of a theme. As I said before this is a good feedback and we will consider it when discussing the future of the theming feature, we have to think if it fits our product strategy before prioritizing and implementing an issue, it doesn’t means that it is wrong. Hope you don’t get me wrong 😊

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
